functorch/functorch/csrc/VmapGeneratedPlumbing.h

```diff
@@ -1564,6 +1564,21 @@ at::Tensor _log_softmax_backward_data_generated_plumbing(const at::Tensor & grad
   return makeBatched(std::get<0>(results), std::get<1>(results), cur_level);
 }
 template <typename batch_rule_t, batch_rule_t batch_rule>
+at::Tensor matrix_power_generated_plumbing(const at::Tensor & self, int64_t n) {
+  c10::impl::ExcludeDispatchKeyGuard guard(kBatchedKey);
+  auto maybe_layer = maybeCurrentDynamicLayer();
+  TORCH_INTERNAL_ASSERT(maybe_layer.has_value());
+  int64_t cur_level = maybe_layer->layerId();
```

Comment:
(no action required) Oh yeah we should make it so that we don't need to check in generated plumbing anymore before folks complain at us about it